openshift/templates/images/nginx/docker/Dockerfile

```
@@ -11,4 +11,6 @@ HEALTHCHECK --start-period=5s CMD pgrep nginx
 
 VOLUME ["/var/www/html", "/var/log/nginx"]
 
+COPY docker/default.conf /etc/nginx/default.conf
```

Shouldn't this be:

```suggestion
COPY default.conf /etc/nginx/default.conf
```

I think currently it would look for a local docker directory to copy from right?

---

I believe @Baelx is right on this.